docs/conf.py

```diff
@@ -18,9 +18,9 @@
 
 # -- Project information -----------------------------------------------------
 
-project = u'RADON'
-copyright = u'2020, RADON team'
-author = u'RADON team'
+project = u'xOpera'
+copyright = u'2020, XLAB d.o.o., xOpera team'
```

Comment:
I believe that the year can be updated to 2021 :smiley:

Reply:
Dammit! :D

Reply:
Here you go, @anzoman! ;)